pytorch_lightning/core/memory.py

```diff
@@ -213,6 +216,29 @@ def out_sizes(self) -> List:
     def param_nums(self) -> List[int]:
         return [layer.num_parameters for layer in self._layer_summary.values()]
 
+    @property
+    def total_parameters(self) -> int:
+        return sum(p.numel() for p in self._model.parameters())
+
+    @property
+    def trainable_parameters(self) -> int:
+        return sum(p.numel() for p in self._model.parameters() if p.requires_grad)
+
+    @property
+    def model_size(self) -> float:
+        return self._model_size()
+    def _model_size(self) -> float:
+        """
+        Estimates total model size i.e total params size in MBs
+        total params size gives model size in accounting total model params.
+
+        NOTE: Currently only Supported total params size.
+
+        Returns:
+            Total estimated model size(MB).
+        """
+        return self.total_parameters * self._precision_megabytes
```

Comment:
can you improve these docs, and merge the two methods together (remove underscore prefix)? There are two incomplete sentences.

Reply:
@awaelchli sure I can, the reason I was separating the function because in follow up PR, i will be adding the other model sizes too 
i.e forward pass + input size

Reply:
@awaelchli _model_size because it will be mostly in private scope as we have given def model_size as public end point via property

Reply:
done